### PR TITLE
nova: Add role and recipe for compute nodes HA

### PR DESCRIPTION
**Pull request by vuntz**
This still needs to be plugged in crowbar, but the code can already be used manually by changing the runlists and some attributes.

### CODE REVIEW

**Comment by mjura** [comment]
Do you want to fix Hound complainings? Especially line it too long :)

**Comment by mjura** [comment]
Besides Hounds bitching change looks OK +1

**Comment by vuntz** [comment]
> Do you want to fix Hound complainings? Especially line it too long :)

The comments about `params ({` lines are incorrect.

I can fix the lines length if needed, though...

**Comment by jsuchome** [comment]
I do not see anything suspicious, so mild +1 from me (for not truly understanding what all is happening here)

**Comment by jsuchome** [comment]
Still +1

File: chef/cookbooks/nova/attributes/default.rb [hunk 1]
```diff
@@ -129,6 +129,17 @@
...
+default[:nova][:ha][:compute][:enabled] = false
+default[:nova][:ha][:compute][:compute][:op][:monitor][:interval] = "10s"
+default[:nova][:ha][:compute][:compute][:op][:monitor][:timeout] = "20s"
+default[:nova][:ha][:compute][:compute][:op][:start][:timeout] = "600s"
+default[:nova][:ha][:compute][:compute][:op][:stop][:timeout] = "300s"
+default[:nova][:ha][:compute][:evacuate][:op][:monitor][:interval] = "10s"
+default[:nova][:ha][:compute][:evacuate][:op][:monitor][:timeout] = "600s"
+default[:nova][:ha][:compute][:evacuate][:op][:start][:timeout] = "20s"
+default[:nova][:ha][:compute][:evacuate][:op][:stop][:timeout] = "20s"
+default[:nova][:ha][:compute][:fence][:op][:monitor][:interval] = "10m"
```
Comment (aplanas) [inline chef/cookbooks/nova/attributes/default.rb]: uninformed question: do this needs some kind of migration?

Comment (vuntz) [inline chef/cookbooks/nova/attributes/default.rb]: No; default attributes are added automatically to the nodes using this cookbook.

**Comment by aplanas** [comment]
+1 (not tested, only read the code)

**Comment by jsuchome** [comment]
+1

**Comment by vuntz** [comment]
Last force-push was just a commit message change, no code change. Gating was green, so merging.

**Pull request merged**